.github/workflows/test-and-deploy-alpha.yml

```diff
@@ -3,27 +3,96 @@ name: test-and-deploy-alpha
 on: [push]
 
 jobs:
-  install-deps:
+  prepare:
     runs-on: [ubuntu-latest]
     steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/restore-cache
-      - uses: ./.github/actions/setup-node
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache node_modules and artifacts
+        uses: ./.github/actions/restore-cache
+
+      - name: Set up Node
+        uses: ./.github/actions/setup-node
+
       - name: Install dependencies
-        run: npm i
-      - name: List all dependencies
-        shell: bash
-        run: npm list
+        run: npm ci --legacy-peer-deps
```

Comment:
Since the package `better-npm-audit` is used for auditing, would it make sense to use the `--no-audit` option here? 
I wonder if it makes it faster

Reply:
Probably yes, we could add that :)